cli/util/api.go

```diff
@@ -274,7 +274,7 @@ func FullNodeProxy[T api.FullNode](ins []T, outstr *api.FullNodeStruct) {
 				}
 
 				total := len(rins)
-				result, err := retry.Retry(ctx, 5, initialBackoff, errorsToRetry, func() (results []reflect.Value, err2 error) {
+				result, _ := retry.Retry(ctx, 5, initialBackoff, errorsToRetry, func() (results []reflect.Value, err2 error) {
```

Comment:
I feel like we should bubble up an error here if we get one? @arajasek thoughts?

Reply:
I investigated this a bit and think it might be the cause of a flaky test. Followup here https://github.com/filecoin-project/lotus/issues/9735